src/controllers/user.controllers.js

```diff
@@ -0,0 +1,41 @@
+import User from '../models/user';
+import { registerValidation, loginValidation } from '../routes/user/validation';
+import { StatusCodes } from 'http-status-codes';
+import bcrypt from 'bcryptjs';
+
+export const createUser = async (data, response) => {
+  const { error } = registerValidation(data);
+  if (error) return response.status(StatusCodes.BAD_REQUEST).send(error.details[0].message);
```

Comment:
It'd probably be better if in case of an error, this endpoint returned a `JSON` as well to make handling responses on the front-end more predictable. This should have at least the key `message` but might probably also need status.